PaillierTests/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="BigInteger" version="1.0.6.32405" targetFramework="net452" />
```

Comment:
PaillierTests references a different version of BigInteger than the main project